controllers/article-controller.js

```diff
@@ -1,16 +1,35 @@
-const { fetchArticlesByID } = require("../models/article-model");
-
+const res = require("express/lib/response");
+const {
+  fetchArticlesByID,
+  fetchArticleVotes,
+} = require("../models/article-model");
 
 exports.getArticlesByID = (req, res, next) => {
-    fetchArticlesByID(req.params)
-      .then((data) => {
-        if (data.length === 0) {
-          res.status(404).send("Not Found");
-        }
-        res.status(200).send({ article: data[0] });
-      })
-      .catch((err) => {
-        next(err);
-      });
-  };
-  
\ No newline at end of file
+  fetchArticlesByID(req.params)
+    .then((data) => {
+      if (data.length === 0) {
+        res.status(404).send("Not Found");
```

Comment:
your code is going to get very repetitive if you keep up this pattern. Take a look at the error handling notes to make use of next